UI/Web/src/app/cards/_modals/bulk-add-to-collection/bulk-add-to-collection.component.html

```diff
@@ -1,46 +1,48 @@
 
-<div class="modal-header">
-    <h4 class="modal-title" id="modal-basic-title">Add to Collection</h4>
-    <button type="button" class="close" aria-label="Close" (click)="close()">
-    <span aria-hidden="true">&times;</span>
-    </button>
-</div>
-<form style="width: 100%" [formGroup]="listForm">
-<div class="modal-body">
-    <div class="form-group" *ngIf="lists.length >= 5">
-        <label for="filter">Filter</label>
-        <div class="input-group">
-            <input id="filter" autocomplete="off" class="form-control" formControlName="filterQuery" type="text" aria-describedby="reset-input">
-            <div class="input-group-append">
-                <button class="btn btn-outline-secondary" type="button" id="reset-input" (click)="listForm.get('filterQuery')?.setValue('');">Clear</button>
+<div id="add-to-collection">
```

Comment:
Don't use ids

Reply:
In addition, modals have a fixed structure, keep the structure as stated, don't wrap with a div. See:
https://ng-bootstrap.github.io/#/components/modal/examples

Reply:
ID removed in abe0d9f